source/_posts/2017-09-14-Droidcon-Berlin-and-GDD-Krakow.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Droidcon Berlin and GDD Krakow
+date: 2017-09-14 17:00:00
+tags: android, conference, architecture, kotlin
+authorIds:
+- joni
+categories:
+- Android
+---
+#Droidcon Berlin
+
+Being one of the biggest Android conferences in Europe, this was my first time in Berlin. The conference was taking part in partnership with IFA Berlin. As a perk Droidcon tickets also allowed access to it. The downside was the big amount of people commuting to the venue and particularly the accommodation prices during those days.
+###Keynote: How to be an Android Expert
+by Chiu-ki Chan
+
+A very eye-opening talk where Chiu-ki shared her views on what an Android expert is and how to become one by sharing knowledge on different ways and tips and tricks to become an Android Expert including how to take sketch notes in a talk or how to write a blog post efficiently. Find the slides [here](https://www.slideshare.net/chiuki/how-to-be-an-android-expert-droidcon-berlin).
+###Data Persistence in Android: There’s Room for Improvement
+by Florina Muntenescu
+
+One of the talks that I was more interested about. Unfortunately, the “Room” in this case was too small for the number of people who wanted to attend so I could not enjoy it this time. Nevertheless, I had another chance as you will see below.
+###Flutter – 60fps UI of the Future
+by Pascal Welsh and Albrecht Noll
+
+[Flutter](https://flutter.io) is a new mobile SDK which aim is to build modern and beautiful UIs for both Android and iOS. Pascal and Albrecht gave an excellent intro to Flutter that even seems capable of running smoothly at 120fps. Impressive!
+Flutter is currently in alpha version and open source with Google working actively on it. It provides a thin layer of C/C++ code and requires to use Dart for implementing the UI. All elements in Flutter are considered widgets that connect in a Matryoshka type hierarchy and uses Composition over Inheritance.
+For more info on the presentation check out the [slides](https://speakerdeck.com/passsy/flutter-60-fps-ui-of-the-future). Also, if you want to try an app build using Flutter, you can try Hamilton ([Android](https://play.google.com/store/apps/details?id=com.hamilton.app), [iOS](https://itunes.apple.com/us/app/hamilton-the-official-app/id1255231054?mt=8)).
+###Home Improvement: Architecture & Kotlin
+by Jorge D. Ortiz-Fuentes
+
+Being Android Architecture Components and Kotlin the two biggest announcements in the last Google I/O this was one of my top pics for day one. Interestingly, I had the chance to meet Jorge early in the morning before the opening keynote.
+During the talk, Jorge shared some insight of real and relevant scenarios where he explained how to improve our apps using Kotlin. Similarly to Florina Muntenescu’s talk, the room was not big enough, but this time I managed to get a sweet spot.
```

Comment:
Do you have enough points on the insights for a separate post?

Reply:
Not really, as I do not know Kotlin I cannot remember the specifics. Could do if the video was available but I can see that it has not been published yet.